tracer/src/Datadog.Trace.Tools.Runner/AnalyzeProcessMemoryCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿// <copyright file="AnalyzeProcessMemoryCommand.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.IO;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Tools.Runner.Checks;
+using Datadog.Trace.Tools.Runner.DumpAnalysis;
+using Spectre.Console;
+using Spectre.Console.Cli;
+using static Datadog.Trace.Tools.Runner.Checks.Resources;
+
+namespace Datadog.Trace.Tools.Runner
+{
+    internal class AnalyzeProcessMemoryCommand : AsyncCommand<AnalyzeProcessMemorySettings>
```

Comment:
You're declaring the command but not actually registering it (so it can't be invoked from the command-line). Is that on purpose?

Reply:
Yeah, I purposefully did not check in that code since this feature is not yet fully baked (the output is a bit messy and is missing the native frames), so I'd prefer users not to see it for now. I plan to add that in the follow-up PR.